test/built-ins/Temporal/PlainDate/prototype/toPlainDateTime/limits.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.toplaindatetime
+description: Numeric bounds check
```

Comment:
I think these tests could have a better description, like "Limits of representable range" (there's nothing particularly numeric about the check)

Reply:
Good spot -- I'll fix the descriptions. Indeed it's not about the limits of numbers qua numbers, but about domain limits. I'll update the descriptions.